Intersect.Server.Core/Migrations/Sqlite/Game/20250322192625_ResourcesStates.cs

```diff
@@ -0,0 +1,209 @@
+using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace Intersect.Server.Migrations.Sqlite.Game
+{
+    /// <inheritdoc />
+    public partial class ResourcesStates : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AddColumn<string>(
+                name: "StatesGraphics",
+                table: "Resources",
+                type: "TEXT",
+                nullable: true);
+
+            var initalId = Guid.NewGuid();
+            var exhaustedId = Guid.NewGuid();
+
+            migrationBuilder.Sql($@"
+                UPDATE Resources
+                SET StatesGraphics = json_object(
+                    '{exhaustedId}', json_object(
+                        'Id', '{exhaustedId}',
+                        'Name', 'Exhausted',
+                        'Texture', Exhausted_Graphic,
+                        'TextureType', Exhausted_GraphicFromTileset,
+                        'RenderBelowEntities', Exhausted_RenderBelowEntities,
+                        'Height', Exhausted_Height,
+                        'Width', Exhausted_Width,
+                        'X', Exhausted_X,
+                        'Y', Exhausted_Y,
+                        'MinimumHealth', 0,
+                        'MaximumHealth', 0
+                    ),
+                    '{initalId}', json_object(
+                        'Id', '{initalId}',
```

Comment:
All of the exhausted and initial states would have the same ID across resources.

While I think this won't cause an immediate issue, if we ever have to separate it into another table it will cause an issue (and we'd have to remember that this issue exists).

On Discord we discussed using invalid placeholder UUIDs for initial/exhausted and then re-assigning invalid IDs in `InitDatabase()`.